core/baseline-core-v1.0-psd01.md

```diff
@@ -2786,113 +2795,729 @@ The document defines storage capability requirements in the following areas:
 
 #### **[R288]**	
 Personal Identifiable Information (PII) MUST NOT be stored in a BPI.
 
+[[R288]](#r288) Testability: 
+
+Preconditions:
+
+* A BPI test environment is installed and configured properly.
+* The BPI storage component contains identification methods to prevent storing Personal Identifiable Information (PII).
+* The BPI Subject has appropriate permissions to access the BPI.
+
+Test Steps:
+
+1. Create a record that contains PII, such as a name, address, social security number, or phone number.
+2. Attempt to store the record in the BPI using a write operation method, such as POST or PUT.
+3. Verify that the BPI rejects the record and returns an error message indicating that the record contains PII and cannot be stored.
+4. Create a record that does not contain PII, such as a random string or number.
+5. Attempt to store the record in the BPI using a write operation method, such as POST or PUT.
+6. Verify that the BPI accepts the record and stores it successfully.
+
+Example Methods to identify PII:
+
+* Use regular expressions or data pattern matching to identify common PII data elements in a record.
+* Compare record data with a pre-defined list of PII data elements to identify PII.
+
+Test Passing Criteria: The test will pass if
+
+* The BPI rejects the storage of records that contain Personal Identifiable Information (PII) and returns an error message indicating that PII cannot be stored.
+* The BPI accepts the storage of records that do not contain PII and stores them successfully.
+* The BPI is able to identify PII using the specified methods.
+* The BPI correctly identifies records that contain PII and rejects their storage.
+
+Note: The test above only verifies write operations, as the requirement is related to the storage of PII. However, read operations should also be checked to ensure that PII cannot be accessed or returned in query results.
+
 #### **[D39]**	
 BPI Storage arranged in a network SHOULD utilize privacy-preserving P2P message protocols.
 
+[[D39]](#d39) Testability: 
+
+Preconditions:
+* BPI Storage is arranged in a two node network that can support pairwise key/identity relationships between storage nodes in a testable manner (see [[R282]](#r282) for testing details)
+
+Test Steps:
+1. Using a pairwise key/identity relationship between the two BPI storage nodes, asymmetrically encrypt a message such as a data replication message.
+2. Perform Test from [[R279]](#r279)
+
+Test Passing Criteria: The test will pass if the test passing criteria in [[R279]](#r279) Testability are met.
+
 ## 7.3. BPI Data Orchestration
+###### BPIDATAORCHESTRATION
+
+Data Orchestration is an automated process for taking siloed data from multiple storage locations, combining and organizing it, and making it available for analysis.
```

Comment:
Given this is defined above in the glossary I'm not sure it needs to be repeated verbatim here.

```suggestion
```

Reply:
No everyone reads the glossary so it is good to have it in the text or at least link to glossary

Reply:
Linking it to the glossary is really good. That's why the web is cool.

Repeating the text 2 or three times is pretty bad. It makes the document "heavier" to read, meaning fewer people will read it carefully.

(And for may readers, it comes across as sloppy editing, which isn't a win).